### PR TITLE
Better Crafting Menu and Better Cargo Console

### PR DESCRIPTION
**Pull request by dwasint**
# About The Pull Request
Ports 
https://github.com/tgstation/tgstation/pull/66933
https://github.com/tgstation/tgstation/pull/65654
https://github.com/tgstation/tgstation/pull/57889 <- only the ui as we had the other code

## Why It's Good For The Game
Better UI makes it easier to navigate

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Better Cargo Console
image

Better Crafting Menu
image

Strippable Menu
image
</details>

## Changelog

:cl:
add:better crafting menu now nicer to look at
add: search bar to supply console
changes:stripable menu is now icon based
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
Would like some screenshots of the changes

**Comment by nednaZ** [comment]
Looks pretty clean.
Will need to test merge it to be sure that everything is functioning properly.

File: tgui/packages/tgui/interfaces/Cargo.js [hunk 1]
```diff
@@ -399,3 +483,58 @@ const CargoCart = (props, context) => {
     </Section>
   );
 };
+
+const CargoHelp = (props, context) => {
+  return (
+    <>
+      <Section title="Department Orders">
```
Comment (nednaZ) [inline tgui/packages/tgui/interfaces/Cargo.js:490]: I don't think we have this feature?

Comment (dwasint) [reply tgui/packages/tgui/interfaces/Cargo.js:490]: we do not i thought our chefs consoles went through that for some reason

**Review by nednaZ — APPROVED** [review]
No issues found or reported.
The menus look cleaner and easier to navigate.

**Pull request merged**